routes/web.php

```diff
@@ -319,34 +333,37 @@
                 $value->isActive = false;
         }
     }
-
-    return Inertia::render('TugasPendahuluan', [
-        'comingFrom' => $comingFrom,
-        'currentUser' => $user,
-        'position' => $position,
-        'userRole' => $userRole->role,
-        'allTP' => $allTP === null ? 'nope' : $allTP,
-        'allModul' => $allModul,
-    ]);
+    $tpPrivilege = array(1,2,15,11,7);
+    if(in_array($userRole->id,$tpPrivilege,true)){
+        return Inertia::render('TugasPendahuluan', [
+            'comingFrom' => $comingFrom,
+            'currentUser' => $user,
+            'position' => $position,
+            'userRole' => $userRole->role,
+            'allTP' => $allTP === null ? 'nope' : $allTP,
+            'allModul' => $allModul,
+        ]);
+    }else return redirect('/');
 })->name('tp')->middleware('loggedIn:asisten');
 
-Route::get('/listTp', function () {
-    $user = Auth::guard('asisten')->user();
-    $userRole = Role::where('id', $user->role_id)->first();
-    $comingFrom = request('comingFrom') === null ? 'none':request('comingFrom');
-    $position = request('position') === null ? 0:request('position');
-    $allKelas = Kelas::all();
-    $allModul = Modul::all();
-
-    return Inertia::render('ListTp', [
-        'comingFrom' => $comingFrom,
-        'currentUser' => $user,
-        'position' => $position,
-        'userRole' => $userRole->role,
-        'allKelas' => $allKelas,
-        'allModul' => $allModul,
-    ]);
-})->name('listTp')->middleware('loggedIn:asisten');
+# DISABLE THIS ROUTE
```

Comment:
No need to comment this whole route, just delete it, we can rollback the code at any point of time